homeassistant/components/fritz/sensor.py

```diff
@@ -119,6 +194,11 @@ def device_class(self) -> str | None:
         """Return device class."""
         return self._sensor_data["device_class"]
 
+    @property
+    def unit_of_measurement(self) -> str | None:
+        """Return unit of measurement."""
+        return self._sensor_data["unit_of_measurement"]
```

Comment:
this can be set using a class attribute in the constructor

```py
self._attr_unit_of_measurement = self._sensor_data["unit_of_measurement"]
````

Reply:
Done, also for other properties